installers/WindowsInstaller.hs

```diff
@@ -96,19 +101,21 @@ fileSubstString :: Text -> Text -> FilePath -> FilePath -> IO ()
 fileSubstString from to src dst =
     TIO.writeFile dst =<< T.replace from to <$> TIO.readFile src
 
-writeInstallerNSIS :: String -> IO ()
-writeInstallerNSIS fullVersion = do
+writeInstallerNSIS :: Version -> Cluster -> IO ()
+writeInstallerNSIS (Version fullVersion') clusterName = do
     tempDir <- fmap fromJust $ lookupEnv "TEMP"
-    let viProductVersion = L.intercalate "." $ parseVersion fullVersion
+    let fullVersion = unpack fullVersion'
+        viProductVersion = L.intercalate "." $ parseVersion fullVersion'
     echo $ unsafeTextToLine $ toText $ "VIProductVersion: " <> viProductVersion
 
     forM_ ["ca.conf", "server.conf", "client.conf"] $
         \f-> fileSubstString "OPENSSL_MD" "sha256" f (f <> ".windows")
 
     writeFile "daedalus.nsi" $ nsis $ do
         _ <- constantStr "Version" (str fullVersion)
+        _ <- constantStr "Cluster" (str $ unpack $ lshowText clusterName)
         name "Daedalus ($Version)"                  -- The name of the installer
-        outFile "daedalus-win64-$Version-installer.exe"           -- Where to produce the installer
+        outFile "daedalus-win64-$Version-$Cluster-installer.exe"           -- Where to produce the installer
```

Comment:
This might break installer uploading logic? /cc @rvl @cleverca22

Reply:
Not for Appveyor, evidently.  But other things, like `iohk-ops` -- sure..

Reply:
So that has to be fixed as well (until we come up with a way to test it on CI)

Reply:
How do you propose we test these changes together?

Reply:
create a development S3 bucket and point uploading scripts there :)

Reply:
It was agreed we should handle this in a separate issue.

Reply:
Meanwhile, let's not make this change. After we have an issue to track the necessary changes across repos to do this we can make focused PRs just for that.

Reply:
Do we have an issue for this @deepfire ?

Reply:
https://iohk.myjetbrains.com/youtrack/issue/DEVOPS-742, @domenkozar